internal/ignition/ignition.go

```diff
@@ -130,6 +130,45 @@ IMAGE=$(echo $1 | sed 's/:.*//')
 podman images | grep $IMAGE || podman rmi --force $1 || true
 `
 
+const okdBinariesOverlayTemplate = `#!/bin/env bash
+set -eux
+# Fetch an image with OKD rpms
+RPMS_IMAGE="%s"
+while ! podman pull --quiet "${RPMS_IMAGE}"
+do
+    echo "Pull failed. Retrying ${RPMS_IMAGE}..."
```

Comment:
Add a small sleep here, just in-case podman fails on something very early, we probably don't want to spin too quickly

Reply:
Fixed in e91b88ece692a523f852439fa2c7fdbf22f88189